waffle/tests/test_templates.py

```diff
@@ -48,19 +48,7 @@ def test_no_request_context(self):
         assert 'switch off' in content
         assert 'sample' in content
 
-    @override_settings(TEMPLATE_LOADERS=(
-        'jingo.Loader',
-        'django.template.loaders.filesystem.Loader',
-        'django.template.loaders.app_directories.Loader',
-    ))
-    @mock.patch.object(template.loader, 'template_source_loaders', None)
-    def test_jingo_tags(self):
-        """
-            We're manually changing default TEMPLATE_LOADERS to enable jingo
-
-            template_source_loaders needs to be patched to None, otherwise
-            TEMPLATE_LOADERS won't be overriden.
-        """
```

Comment:
I don't think I understand this code. It looks like in the old and new versions `TEMPLATE_LOADERS` is set in `test_settings.py`. So I don't think this is actually overriding any settings.

Does that suggest this test isn't valid?